packages/popover/src/positionUtils.spec.ts

```diff
@@ -2581,12 +2852,28 @@ describe('positionUtils', () => {
     });
   });
 
-  describe('getElementPosition', () => {
+  describe('getElementDocumentPosition', () => {
+    test('given an element, it returns an object with information about its position', () => {
+      const div = document.createElement('div');
```

Comment:
Should we set some initial information about the document to test this better? Like we do for the above tests?

Reply:
Potentially, though this is a direct port of what was already here. I was gonna start making some test improvements, but I started going down a rabbit-hole. 

For the time being, my goal here was simply to maintain the existing test coverage rather than expand it.